bin/inference/pycbc_inference

```diff
@@ -151,6 +151,15 @@ with ctx:
     # construct class that will return the natural logarithm of likelihood
     model = models.read_from_config(cp, **model_args)
 
+    # Check if lower frequency cutoff on the command line and in config file are the same
+    if model._f_lower != low_frequency_cutoff_dict.values()[0]:
```

Comment:
@cdcapano @stephanmohr Was looking at the changes in this PR. Is a low-frequency-cutoff being specified in the command line as well as the config file? What's the difference between the two?